### ISSUE TITLE
FEATURE: Sync DNS servers

### ISSUE THREAD

**Issue by 0xnim** [face-hh/webx#175]
**Is your feature request related to a problem? Please describe.**
The current webxDNS system can only be run by itself.

**Describe the solution you'd like**
There should be a sync between webxDNS servers. 
The solution that I came up from the top of my head is by having something like Zone files and syncing those from the main server. 
This would also allow subdomains and other features to be implemented easily.

**Describe alternatives you've considered**
Another solution which is easier but not good for the long term. Just having DNS servers request /domains until they get all of them, and do this every x hrs or something.

**Additional context**
I will likely implement this.

**Comment by KubaPro010** [comment]
There's `/domains`

**Comment by KubaPro010** [comment]
The best and easiest way to do it live would be websockets.

**Comment by 0xnim** [comment]
Why do you think websockets would be a good alternative?

**Comment by 0xnim** [comment]
This PR Implements it using the alternative way.

**Comment by KubaPro010** [comment]
> Why do you think websockets would be a good alternative?

because they're a full-duplex conection, meaning you can send anything live with them, and if you lose connection then you could go to `/domains`

**Comment by RealPacket** [comment]
> > Why do you think websockets would be a good alternative?
> 
> because they're a full-duplex conection, meaning you can send anything live with them, and if you lose connection then you could go to `/domains`

Yes, so if we have a new domain for a certain DNS server, we could send an event:
```json
{
  "type": "new_domain",
  "info": ...
```
